handlers/handlers.go

```diff
@@ -158,6 +158,9 @@ func censusLanding(ctx context.Context, w http.ResponseWriter, req *http.Request
 	var initialVersion dataset.Version
 	var initialVersionReleaseDate string
 	var err error
+	var form = req.URL.Query().Get("f")
```

Comment:
I assume the hidden input is linked to this but hmmmm

Reply:
Yeah, not creatively named though. Just checking that it's the right form